packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -123,11 +124,52 @@ class CupertinoCheckbox extends StatefulWidget {
 
   /// The color to use when this checkbox is checked.
   ///
+  /// If [fillColor] returns a non-null color in the [WidgetState.selected]
+  /// state, it will be used instead of this color.
```

Comment:
```suggestion
  /// state, [fillColor] will be used instead of [activeColor].
```

I was a little confused at first glance. Maybe this reduces ambiguity a bit.